src/ml-service/packages/sensor_location/app/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Normally **.vscode** is added to gitignore file to prevent your local settings getting exposed to the world